### ISSUE TITLE
Remove the folder mapping to a local folder in the test definition

### ISSUE THREAD

**Issue by VincenzoFerme** [benchflow/benchflow#296]
We do not need anymore the folder to be exposed to a local folder, after https://github.com/benchflow/benchflow/issues/254

Consequently the same change has to be applied to the `wercker.yml` file in https://github.com/benchflow/benchflow/blob/devel/wercker.yml#L122 and https://github.com/benchflow/benchflow/blob/devel/wercker.yml#L171

**Comment by jeppe-style** [comment]
Yes, this is only needed if we want to preserve the data after the mongo container is stopped.
